grow/documents/document.py

```diff
@@ -1,11 +1,11 @@
 """Grow documents."""
 
-import datetime
 import json
 import logging
 import os
 import re
 import yaml
+from slugify import slugify
```

Comment:
If we're following the google style guide, we should import the module here (`import slugify`) instead of the function within the module (`from slugify import slugify`)

Reply:
Updated to use just `import slugify`